code/game/objects/items/weapons/blades.dm

```diff
@@ -28,8 +28,9 @@
 	force = 60
 	attack_speed = 12
 	w_class = WEIGHT_CLASS_BULKY
-	flags_item = DRAINS_XENO
+	flags_item = DRAINS_XENO|TWOHANDED
```

Comment:
undocumented? 
Also, wasn't vali only slighlty viable with a shield? I don't think bumpslash will make it sudendly OP

Reply:
This is required to allow harvester to be wielded.

Reply:
Why are you requiring it to be wielded for the bump attack? Is it a design intent or because you didn't find a way to do it without?

Reply:
It's a design intent so people don't use shield + vali and such.

Reply:
Eh, your choice, I don't see why you keep making vali a weak setup